test/release/rollup-ts.sh

```diff
@@ -15,7 +15,7 @@ minify_in_situ() {
     for i in `find $DIRS -type f -name '*.ts' | grep -v '\.d\.ts'`
     do
         echo "$i"
-        node_modules/.bin/esbuild --loader=ts --target=node14 < "$i" \
+        node_modules/.bin/esbuild --loader=ts --target=es2019 < "$i" \
```

Comment:
`--target=es2020` is needed to produce `ChainExpression`s, otherwise they are down-leveled to ES2019.

Reply:
`?.` aren't supported by `uglify-js` yet.

Reply:
Also we'll probably need es2021 when we eventually clear that since otherwise private class prorperties won't get through (no idea if `esbuild` does that, but that's how `acorn` works...)

Reply:
esbuild handles everything in the ES standard, Typescript and likely to be approved stage 3 proposals already implemented by Chrome, including private class properties. esbuild would have a different set of generated helper functions than Babel or tsc produces for down-leveling to earlier versions of ES.

Reply:
```
$ esbuild --version
0.11.14
```
```
$ echo 'class C { #p; static #s; }' | esbuild --target=es2020
var _p, _s;
class C {
  constructor() {
    _p.set(this, void 0);
  }
}
_p = new WeakMap();
_s = new WeakMap();
_s.set(C, void 0);
```
```
$ echo 'class C { #p; static #s; }' | esbuild --target=esnext
class C {
  #p;
  static #s;
}
```